src/Mafia/Init.hs

```diff
@@ -150,18 +154,16 @@ data Profiling =
   | EnableProfiling
     deriving (Eq, Ord, Show)
 
-data InstallState =
-  InstallState {
-      isSourceDependencies :: Set SourcePackage
-    , _isCabalFile         :: Hash
-    } deriving (Eq, Ord, Show)
-
 data MafiaState =
   MafiaState {
-      msInstallState :: InstallState
```

Comment:
Any reason for this change apart from cleaning up? Looks like the json file has the same information?